catalog/admin/module/catalog-admin-module-sources/src/main/webapp/templates/deleteSource.handlebars

```diff
@@ -11,7 +11,10 @@
  *
  **/
  --}}
-<div class="delete-source-name">{{name}}</div>
+<div class="delete-source-name"><span style="margin-right: 2em">{{name}}</span>
```

Comment:
It'd be nice to make this look slightly nicer, but I'm not sure what the desired look would be.